coalib/bears/Bear.py

```diff
@@ -20,10 +20,29 @@
 from coalib.settings.FunctionMetadata import FunctionMetadata
 from coalib.settings.Section import Section
 from coalib.settings.ConfigurationGathering import get_config_directory
+from coalib.settings.Setting import typed_list
 
 from .meta import bearclass
 
 
+def _debugger_flag_setter(bear):
```

Comment:
this is not a "setter".  It doesnt set anything.

It determines the value of the `debugger` *setting*.

Reply:
>It determines the value of the debugger setting.

No its not this is not the way current code is working.

Does your lower comment of `typed_list` is because of the above confusion?

Reply:
Its only setting the `debugger` flag to `True` or `False` depending the cases and after that this falg is used to enable debugger.